### ISSUE TITLE
Import MapLocal settings did not restore my local file map path

### ISSUE THREAD

**Issue by rxg9527** [ProxymanApp/Proxyman#1264]
### Proxyman version? (Ex. Proxyman 1.4.3)
Version 3.6.0 (30600)

### macOS Version? (Ex. mac 10.14)
12.2.1

### Steps to reproduce 
I have 2 commonly used Macs for development, both have the same path directory and response JSON files for MapLocal, and will use Git and iCloud to keep in sync between 2 Macs.
I tried several versions of Proxyman and found that the logic of Proxyman exporting MapLocal is like this: after exporting MapLocal settings on mac A, importing them on mac B, the local file path of Map from will be replaced with `~/Library/Application Support/com.proxyman.NSProxy/map-local/import/xxxx`, instead of the original file path I configured myself.

### Expected behavior
I hope to give an import option for MapLocal that will allow me to choose to use my own configured file path.

### Screenshots (optional)
<img width="1160" alt="Snipaste_2022-06-24_12-28-10" src="https://user-images.githubusercontent.com/11425983/175462373-3057e59a-e957-4abe-abe6-3dd3b7eeb22c.png">

**Comment by NghiaTranUIT** [comment]
Yes, it's the feature. Many users would share the Map Local Rule with their colleagues (on different Mac machines), so the Map Local should include the Map Local File too.

To workaround, after importing a new rule, you can re-select the local file (the one on iCloud), by clicking on the "Select Local File" button.

---------------

Meanwhile, I will reconsider to add the option to opt-out this behavior 👍

**Comment by rxg9527** [comment]
To workaround, can I accomplish this by Git the `~/Library/Application Support/com.proxyman.NSProxy` folder on Mac a and synchronizing on Mac b？

**Comment by NghiaTranUIT** [comment]
yes, it should work @rxg9527. All Proxyman configs are stored in `~/Library/Application Support/com.proxyman.NSProxy`. Make sure two machines has the same Proxyman version 👍
